tests/e2e/grpcapp/grpc_test.go

```diff
@@ -0,0 +1,72 @@
+// +build e2e
```

Comment:
Pls remove this file. This code is testing your testapp, not daprd. 

You can call testapp http endpoint and this test app needs to call grpc api. so that you can reuse the existing go test.

1. for http api testing, you can call http test http restful test api to stateapp.
2. for grpc api testing, you can call grpc test http restful test api to stateapp.

You can use the same test code and same test data using the different http restful test api.